src/components/formik-radiobutton/FormikRadiobutton.tsx

```diff
@@ -75,12 +60,12 @@ const RoundStyled = styled.div<{ error?: boolean, disabled?: boolean }>`
 `;
 
 const FillingStyled = styled.span<{ error?: boolean, disabled?: boolean }>`
-  display: inline-block;
+  display: block;
   margin: 2px;
-  width: 15px;
-  height: 15px;
+  width: 1.1em;
+  height: 1.1em;
```

Comment:
Kun je hier niet ook met variabelen werken ?

Reply:
Dan gaan de radiobuttons te drastisch van grootte veranderen vrees ik. Reden waarom ik geswitcht ben naar em's is dat de radiobuttons meegroeien met de font-size.

Reply:
Oké